pkg/kudoctl/packages/verifier/types.go

```diff
@@ -0,0 +1,26 @@
+package verifier
+
+import (
+	"fmt"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/packages"
+)
+
+type ParamWarning string
```

Comment:
Why is this string type-renaming needed? And can't e.g. a `ParamError` simply be of type `Error`?

Reply:
I see `verifier.Error` and `Errors` as completely semantically different than Go Errors.   These are not control flow errors... these are linter errors.   Linter errors are a collection of things wrong with the operator not an error to be handled IMO